Compiler/SimCode/SimCodeMain.mo

```diff
@@ -778,6 +795,7 @@ algorithm
       Boolean isFMI2;
       String fmiVersion;
       BackendDAE.SymbolicJacobians fmiDer;
+      Boolean symJacBackup;
```

Comment:
`symJacBackup` is also never used

Reply:
Seems you are right.
@niklwors You added it in commit https://github.com/niklwors/OMCompiler/commit/d835ce5ccdbd9f18e8dd7c898bc389936fe9839a
I think we just forgot to delete it...